packages/tools/devtools/devtools-example/README.md

```diff
@@ -18,12 +18,6 @@ You can run this example using the following steps:
     - For an even faster build, you can add the package name to the build command, like this:
       `pnpm run build:fast --nolint @fluid-example/devtools-example`
 1. Run `pnpm start` from this directory and open <http://localhost:8080> in a web browser to see the app running.
-1. This package uses pm2 to setup a cross-package hot reloading solution. When you run `pnpm run start` it will start multiple processes with pm2. pm2 is configured using the [pm2 ecosystem file](./pm2.ecosystem.config.js) and [pm2's offical documentation](https://pm2.keymetrics.io/docs/usage/quick-start/). Here are a few helpful commands:
```

Comment:
We'll want to move this out of the auto-generated section for this package, it should stay.

Reply:
Good catch!! Thanks!